src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
1. it's a good practice to use object destructing.
2. you can move status check to a variable for better readability
```suggestion
export const Person = ({ person }) => {
  const {...} = person;
  const partnerStatus = sex === 'm' ? 'wife' : 'husband';
  
  return (
    <section className="Person">
```